packages/visualizations/src/components/Map/WebGl/constants.ts

```diff
@@ -25,24 +23,35 @@ export const DEFAULT_SORT_KEY_VALUE: DataDrivenPropertyValueSpecification<number
     0,
 ];
 
+export const POPUP_DISPLAY = {
+    tooltip: 'tooltip',
+    sidebar: 'sidebar',
+    modal: 'modal',
+} as const;
+
 export const POPUP_WIDTH = 300;
 
 export const CONTROL_POSITION: ControlPosition = 'top-right';
 
 // Update styles in ./MapRender.svelte if one of these classnames must change.
-const POPUP_CLASSNAME = 'poi-map__popup';
-export const POPUP_FEATURE_CONTENT = 'poi-map__popup-feature-content';
-export const POPUP_FEATURE_CONTENT_LOADING = 'poi-map__popup-feature-content--loading';
-export const POPUP_NAVIGATION_CONTROLS_CLASSNAME = 'poi-map__popup-navigation-controls';
+const POPUP_CLASSNAME = 'ods-visualization__map-popup';
```

Comment:
Since we're beta, the API change seems fair enough. Not sure it's fully needed though, we have an `ods-visualization-poi-map` that wraps everything (but maybe the popup is out of this tree?).

Reply:
Adding classes to the popup is required to handle our 3 variants (tooltip, sidebar and modal)

However, now the map canvas is wrapper around a container (`.ods-visualization__map-container`)
So we can use it to scope our style. 

Instead of doing 
`:global(.ods-visualization__map-popup.ods-visualization__map-popup--as-sidebar)`
we can do 
`.ods-visualization__map-container :global(.maplibregl-popup--as-sidebar),`

See https://github.com/opendatasoft/ods-dataviz-sdk/pull/245/commits/5b42d5d118b92486cf86c288d4eb45e9efd6120b

It limit a bit the usage of custom classnames